nemo/collections/nlp/models/language_modeling/megatron_gpt_model.py

```diff
@@ -395,6 +395,11 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
 
         self.inference_params = None
 
+        # Reset learning rate params
+        self.if_init_step = True
```

Comment:
any difference between self.first_step vs. this ?

Reply:
> any difference between self.first_step vs. this ?

hmm that's a good point
I think I can try to use one variable for both of places